config/locales/en/providers.yml

```diff
@@ -472,6 +472,24 @@ en:
         feedback_suffix: " to help us improve this service."
         view_completed_application: View completed application
         back_to_your_applications: Back to your applications
+    bank_statements:
+      show:
+        h1-heading: Upload bank statements
+        info: We need bank statements from the last 3 months for all of your client's bank accounts.
+        h2-heading: Uploaded files
+        no_files: Files uploaded will appear here.
+        upload_file: Upload files
+        size_hint: The maximum file size is 7MB. Files must be a DOC, DOCX, RTF, ODT, JPG, BMP, PNG, TIF or PDF.
```

Comment:
Prototype just lists the max file size and does not indicate file types

Reply:
question for design - and should it applied consistently